lib/brightcove/cmsapi.rb

```diff
@@ -2,7 +2,6 @@
 
 module Brightcove
   class Cmsapi
-    VERSION = "0.1.0"
```

Comment:
What was wrong with this approach?

Reply:
It was breaking CI https://travis-ci.org/nhsuk/brightcove-cmsapi/builds/270483504?utm_source=github_status&utm_medium=notification

Coulnd not work out how to fix, so it seemed like best solution to move it to gemspec.